README.md

```diff
@@ -513,7 +513,20 @@ MessageDb support is provided in the samples and the `eqx` tool:
 - being able to supply `mdb` flag to `eqx dump`, e.g. `eqx dump -CU "Favorites-ab25cc9f24464d39939000aeb37ea11a" mdb -c "pgconnectionstring"`
 - being able to supply `mdb` flag to Web sample, e.g. `dotnet run --project samples/Web/ -- mdb -c "pgconnectionstring"`
 
-Equinox does not provide utilities for configuring or installing message-db. See [MessageDB's installation documentation](http://docs.eventide-project.org/user-guide/message-db/install.html).
+Equinox does not provide utilities for configuring or installing MessageDB. See [MessageDB's installation documentation](http://docs.eventide-project.org/user-guide/message-db/install.html).
+
+In addition to the default access strategy of reading the whole stream forwards in batches, the following access strategies are supported in MessageDb:
+
+`AccesStrategy.LatestKnownEvent`
+- Uses message-db's `get_last_stream_message` API to only ever fetch the last event in a stream
+- This is useful for aggregates whose entire state can be constructed from the latest event (e.g. a stream that stores checkpoints)
+- NOTE: The last event MUST be decodable by the supplied codec. Otherwise the stream is assumed to be empty
```

Comment:
should be mentioned in the xmldoc if not yet

Reply:
Clarified that it's not empty, but that it's only a single event, and therefore you'll get the initial state if the last event isn't decodable. MUST -> should